src/core/modules/impl/HandlerExecutorFactory.ts

```diff
@@ -197,7 +197,8 @@ export class HandlerExecutorFactory implements ExecutorFactory<HandlerApi<unknow
               if (!cond) throw new ContractError(message);
             },
             //https://github.com/patriksimek/vm2/issues/484#issuecomment-1327479592
-            Uint8Array: Uint8Array
+            Uint8Array: Uint8Array,
```

Comment:
I guess we should add all types from https://developer.mozilla.org/en-US/docs/Web/JavaScript/Reference/Global_Objects#indexed_collections ...